docs/tutorials/g3-algebra-of-space.ipynb

```diff
@@ -39,11 +39,12 @@
   },
```

Comment:
<p>I think it would be better to just change this box to:</p><pre class="ql-syntax" spellcheck="false">from math import cos, sin, pi

theta = pi/4
R = cos(theta) - sin(theta)*e23
R
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pygae/clifford/pull/342/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
(this comment only makes sense on ReviewNB it seems...)